Dockerfile

```diff
@@ -1,20 +1,16 @@
-FROM golang:1.7.1-onbuild
+FROM golang AS build
 
```

Comment:
I agree with not using the `-onbuild` image, but let's be specific about which version of Golang we're using.